src/Microsoft.IdentityModel.Tokens/SecurityTokenDescriptor.cs

```diff
@@ -52,7 +52,7 @@ public class SecurityTokenDescriptor
         public EncryptingCredentials EncryptingCredentials { get; set; }
 
         /// <summary>
-        /// Gets or sets the value of the 'expiration' claim.
+        /// Gets or sets the value of the 'expiration' claim. This value should be in UTC.
```

Comment:
>This value should be in UTC. [](start = 62, length = 28)

consider adding an aka.ms link to the wiki page you created

Reply:
[https://aka.ms/msal-net-on-behalf-of](https://aka.ms/msal-net-on-behalf-of)

---
In reply to: [336267198](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1279#discussion_r336267198) [](ancestors = 336267198)

Reply:
I created an issue to track this: #1283